generative_ai/gemini_system_instruction.py

```diff
@@ -24,6 +24,7 @@ def set_system_instruction(project_id: str) -> str:
 
     vertexai.init(project=project_id, location="us-central1")
 
+    # TODO(developer): Check latest AI models at https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/inference
```

Comment:
Let's not put the TODO here. The page where the sample is shown should have the model information.